### PR TITLE
Add pygments-anyscript package

### PR DESCRIPTION
**Pull request by melund**
This adds the pygments-anyscript packages. A pygments lexer and style to highlight the AnyScript modeling language.

### CODE REVIEW

**Comment by melund** [comment]
@conda-forge/core 
The PyPi urls doesn't seem to work at the moment. Is this a problem you have seen before

The package lives here on pypi:
https://pypi.python.org/pypi/pygments-anyscript

but the URL constructed in the `meta.yml` [does not work](https://travis-ci.org/conda-forge/staged-recipes/builds/260080438#L227): 
https://pypi.io/packages/source/p/pygments-anyscript/pygments-anyscript-0.2.0.tar.gz

**Comment by isuruf** [comment]
Url should be https://pypi.io/packages/source/p/pygments_anyscript/pygments_anyscript-0.2.0.tar.gz

**Comment by melund** [comment]
Thanks @isuruf. I mixed up the project and package name. I think all issues are fixed now.

File: recipes/pygments-anyscript/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/pygments-anyscript/meta.yaml]: @melund please remove all the boilerplate comments from the example.

**Comment by melund** [comment]
@ocefpaf I thought it would be nice to at least leave some comments in the file for my co-maintainers, who have never worked with conda-recipes before. But it is not a big deal. 

I have removed all comments. It should be ready now.

**Comment by ocefpaf** [comment]
> I thought it would be nice to at least leave some comments in the file for my co-maintainers

We have the docs for that and, after the recipe is created, there is very little that needs to be changed. (checksum, build and version numbers.)

**Pull request merged**